babel/messages/frontend.py

```diff
@@ -801,6 +821,18 @@ def run(self):
                 shutil.copy(tmpname, filename)
                 os.remove(tmpname)
 
+        if self.check:
+            for filename, up_to_date in check_status.items():
+                if up_to_date:
+                    self.log.info('Catalog %s is up to date.', filename)
+                else:
+                    self.log.warning('Catalog %s is out of date.', filename)
+            if not all(check_status.values()):
+                raise DistutilsError("Some catalogs are out of date.")
```

Comment:
This results unfortunately in a traceback presented to the user. There seems to be no established way of providing an exit code to the user at the moment. Let me know if you want me to look into it.

Reply:
That's probably fine – #824 and all, we'll probably ditch the Distutils-ness compat sooner or later.